requirements.txt

```
@@ -1,8 +1,8 @@
 numpy>=1.0
 scipy>=1.0
 matplotlib>=3.0
-netCDF4>=1.4,<1.6; python_version == "3.7"
-netCDF4>=1.6.2; python_version != "3.7"
+netCDF4>=1.4,<1.6; python_version < "3.11"
+netCDF4>=1.6.2; python_version >= "3.11"
 windrose>=1.6.8
```

The correct fix would be:


```suggestion
netCDF4>=1.4; python_version < "3.11"
netCDF4>=1.6.2; python_version >= "3.11"
windrose>=1.6.8
```